src/reducers.js

```diff
@@ -7,11 +8,22 @@ import { searchPageReducer } from 'components/SearchPage/SearchPageReducer';
 import { uploadsReducer } from 'components/Uploads/UploadsReducer';
 import { noteDetailsReducer } from 'components/NoteDetails/NoteDetailsReducer';
 
-export const reducers = combineReducers({
-  app: appReducer,
-  notes: notesReducer,
-  noteDetails: noteDetailsReducer,
-  notesMindMap: notesMindMapReducer,
-  searchPage: searchPageReducer,
-  [uploadsReducer.KEY]: uploadsReducer,
-});
+// export const reducers = combineReducers({
+//   app: appReducer,
+//   notes: notesReducer,
+//   noteDetails: noteDetailsReducer,
+//   notesMindMap: notesMindMapReducer,
+//   searchPage: searchPageReducer,
+//   [uploadsReducer.KEY]: uploadsReducer,
+// });
```

Comment:
Please delete commented code.

Reply:
I need it untill redux, saga are completely refactored